ansible/inventory.yml

```diff
@@ -129,20 +129,17 @@ hosts:
           aix72-ppc64-2: {ip: 140.211.9.36, description: p8-aix2-ojdk04.osuosl.org}
           centos74-ppc64le-1: {ip: 140.211.168.228, user: centos}
           centos74-ppc64le-2: {ip: 140.211.168.217, user: centos}
-          centos74-ppc64le-4: {ip: 140.211.168.245, user: centos}
```

Comment:
was that meant to be removed?

Reply:
Yes - it doesn't exist - spotted it while removing the Ubuntu systems - will add comment to the PR description

Reply:
ping @karianna as you're currently blocking this

Reply:
ping @karianna - if you can approve and marge that would be appreciated so that the current state of these machines is close to reality during the time I'm away